msyncd/AccountsHelper.cpp

```diff
@@ -100,7 +100,6 @@ void AccountsHelper::slotAccountCreated(Accounts::AccountId id)
                 delete syncProfile;
             }
         }
-        delete newAccount;
     }
```

Comment:
I just double checked this, and it turns out that this code already keeps one instance of each account in memory at all times anyway (as it listens to every account for changes which might affect scheduling).  So, the code as it stands in this PR will actually be more optimal than attempting to manage the accounts manually, as it means that we won't have "duplicate" instances created in memory for short periods of time.